tests/integration/helpers/proxy1.py

```diff
@@ -0,0 +1,80 @@
+import socket
+import threading
+
+
+class Proxy1:
+    def __init__(self, proxy_string=""):
+        self.proxy_string = proxy_string
```

Comment:
probably it's better to add a special constructor to avoid raw string, but this one is also needed if we want for example to test something with broken PROXY protocol itself
but it's just ad-hoc class to test PROXY processing, and any other features can be added in the future when needed